src/databricks/labs/ucx/source_code/dbfs.py

```diff
@@ -65,14 +65,19 @@ class DBFSUsageLinter(Linter):
     def __init__(self):
         pass
 
+    @property
```

Comment:
I don't understand the use of this property

Reply:
This is no longer present with the next push